cmd/grpc-health-probe/main.go

```diff
@@ -89,49 +91,49 @@ func main() {
 	opts = append(opts, grpc.WithInsecure())
 
 	if verbose {
-		log.Print("establishing connection")
+		log.Info("establishing connection")
 	}
 	connStart := time.Now()
 	dialCtx, cancel2 := context.WithTimeout(ctx, connTimeoutDur)
 	defer cancel2()
 	conn, err := grpc.DialContext(dialCtx, remoteURL, opts...)
 	if err != nil {
 		if err == context.DeadlineExceeded {
-			log.Printf("timeout: failed to connect service %q within %v", remoteURL, connTimeoutDur)
+			log.Infof("timeout: failed to connect service %q within %v", remoteURL, connTimeoutDur)
 		} else {
-			log.Printf("error: failed to connect service at %q: %+v", remoteURL, err)
+			log.Infof("error: failed to connect service at %q: %+v", remoteURL, err)
 		}
 		os.Exit(StatusConnectionFailure)
 	}
 	connDuration := time.Since(connStart)
 	defer conn.Close()
 	if verbose {
-		log.Printf("connection establisted (took %v)", connDuration)
+		log.Infof("connection established (took %v)", connDuration)
 	}
 
 	rpcStart := time.Now()
 	rpcCtx, rpcCancel := context.WithTimeout(ctx, rpcTimeoutDur)
 	defer rpcCancel()
 	resp, err := healthpb.NewHealthClient(conn).Check(rpcCtx, &healthpb.HealthCheckRequest{Service: serviceName})
-	log.Print(resp)
+	log.Infof("response %q", resp)
```

Comment:
Pretty sure this is not a useful log message, really. Recommend removing it.

Reply:
Ack.